### PR TITLE
frontend/feature/log-events

### PR DESCRIPTION
**Pull request by SanskarGubreley**
## Type of Change

- [ ] Bugfix
- [x ] New feature
- [x ] Enhancement
- [ x] Refactoring
- [ ] Dependency updates

### Additional Changes

- [ ] This PR modifies the database schema (database migration added)
- [ ] This PR modifies dhall configs/environment variables
- [ ] This PR contains API breaking changes

## Checklist

- [ ] I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable

### CODE REVIEW

File: Frontend/ui-common/src/Engineering/Helpers/LogEvent.purs [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+foreign import getLogDestination :: Effect (Array String)
+
+isElement :: String -> Array String -> Boolean
+isElement logBrand logBrands = isJust $ find (\el -> el == logBrand) logBrands
+
+triggerLog :: String -> Object Foreign -> String -> Effect Unit 
+triggerLog event logField logDestination = case logDestination of  
+  "JUSPAY" -> do 
+    _ <- pure $ spy "second one" " "
```
Comment (kavya-shree-s) [inline Frontend/ui-common/src/Engineering/Helpers/LogEvent.purs:30]: @SanskarGubreley Remove log lines

File: Frontend/ui-common/src/Engineering/Helpers/LogEvent.purs [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    _ <- firebaseLogEvent event 
+    pure unit 
+  "FIREBASE" -> do 
+    let 
+      eventObject = insert "event" (encode event) empty
+      foreignObject = rootLevelKeyWithRefId logField
+    _ <- trackActionObject Tracker.User Tracker.Info ON_EVENT eventObject foreignObject
+    pure unit 
+  _ -> do 
+    _ <- pure $ spy  "heghfvj" ""
```
Comment (kavya-shree-s) [inline Frontend/ui-common/src/Engineering/Helpers/LogEvent.purs:40]: @SanskarGubreley don't commit log , If require add proper log

**Review by kavya-shree-s — CHANGES_REQUESTED** [review]

File: Frontend/ui-common/src/Engineering/Helpers/LogEvent.purs [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    _ <- trackActionObject Tracker.User Tracker.Info ON_EVENT eventObject foreignObject
+    pure unit 
+  _ -> do 
+    _ <- pure $ spy  "heghfvj" ""   
+    pure unit 
+
+logEvent :: Object Foreign -> String -> Effect Unit 
+logEvent logField event = do 
+  logDestinations <- getLogDestination 
+  _ <- pure $ spy "first one" ""
```
Comment (kavya-shree-s) [inline Frontend/ui-common/src/Engineering/Helpers/LogEvent.purs:46]: @SanskarGubreley Remove log lines

**Review by kavya-shree-s — CHANGES_REQUESTED** [review]

File: Frontend/ui-common/src/Engineering/Helpers/LogEvent.purs (hunk 2 above)
Comment (SanskarGubreley) [reply Frontend/ui-common/src/Engineering/Helpers/LogEvent.purs]: will remember it.

**Pull request closed without merging**